lib/jsdom/living/generator/events/EventTarget.idl

```diff
@@ -0,0 +1,6 @@
+[Exposed=(Window,Worker)]
```

Comment:
Wow, sorry, I have no idea what I was thinking when I said that. Probably either living/generated/events or living/events? My bad.